gradlew.bat

```diff
@@ -14,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"
 
```

Comment:
We definitely should add more memory.
This is some ugly gradle behavior that they changed memory to 64m

Reply:
This is `gradlew.bat` distributed by Gradle Inc. This are the options for JabRef compilation. I did not notice any difficulties when compiling JabRef with the *default* distributed by Gradle Inc. Did you?

In other words: This is not JabRef's setting.

In case, there are difficulties, we should set `GRADLE_OPTS` (as indicated at the comment) since I don't want to patch a generated file.